sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,242 @@
+# July'21 release notes
+
+Release notes for commit range 6a49170027fb..962909fe9e78
+
+## New features
+ - Implemented SYCL 2020 specialization constants [07b27965] [ba3d657]
+   [bd8dcf4] [d15b841]
+ - Provided SYCL 2020 function objects [24a2ad89]
+ - Added support for ITT notification in SYCL Runtime [a7b8daf] [8d3921e3]
+ - Implemented SYCL 2020 sub_group algorithms [e8caf6c3]
+ - Implemented SYCL 2020 `sycl::handler::host_task` method [75e5a269]
+ - Implemented SYCL 2020 `sycl::sub_group` class [19dcac79]
+ - Added support for AMD GPU devices [ec612228]
+ - Implemented SYCL 2020 `sycl::is_device_copyable` type trait [44c1cbcd]
+ - Implemented SYCL 2020 USM features [1df6873d]
+ - Implemented support for Device UUID from [Intel's Extensions for Device Information](doc/extensions/IntelGPU/IntelGPUDeviceInfo.md) [25aee287]
+ - Implemented SYCL 2020 `sycl::atomic_fence` [dcd59547]
+ - Implemented `intel::loop_count_max`, `intel::loop_count_max`,
+   `intel::loop_count_avg` attributes that allow to specify number of loop
+   iterations for FPGA [f74b4ef]
+ - Implemented generation of compiler report for kernel arguments [201f902]
+ - Implemented SYCL 2020 `[[reqd_sub_group_size]]` attribute [347e41c]
+ - Implemented support for `[[intel::named_sub_group_size(primary)]]` attribute
+   from [sub-group extension](doc/extensions/SubGroup/SYCL_INTEL_sub_group.asciidoc#attributes)
+   [347e41c]
+ - Implemented SYCL 2020 interoperability API [e6733e4]
+ - Added [group sorting algorithm](doc/extensions/GroupAlgorithms/SYCL_INTEL_group_sort.asciidoc)
+   extension specification [edaee9b]
+ - Added [initial draft](doc/extensions/LevelZeroBackend/LevelZeroBackend.md)
+   for querying of free device memory in LevelZero backend extension [fa428bf]
+ - Added [InvokeSIMD](doc/extensions/InvokeSIMD/InvokeSIMD.asciidoc) and
+   [Uniform](doc/extensions/Uniform/Uniform.asciidoc) extensions [72e1611]
+ - Added [Matrix Programming Extension for DPC++ document](doc/extensions/Matrix/dpcpp-joint-matrix.asciidoc) [ace4c733]
+ - Implemented SYCL 2020 `sycl::span` [9356d53]
+ - Added [device-if](doc/extensions/DeviceIf/device_if.asciidoc) extension
+   [4fb95fc]
+ - Added a [programming guide](doc/MultiTileCardWithLevelZero.md) for
+   multi-tile and multi-card under Level Zero backend [d581178a]
+
+## Improvements
+### SYCL Compiler
+ - Use `opencl-aot` instead of `aoc` when AOT flow for FPGA Emulator is
+   triggered [3a99558]
+ - Allowed for using an external host compiler [6f0ad1a]
+ - Cleaned up preprocessing output when `-fsycl` option is passed [3a18db6]
+ - Allowed kernel names in anonymous namespace [e47dbad]
+ - Set default value of `-sycl-std` to `2020` when `-fsycl-is-device` or
+   `-fsycl-is-host` specified [680adc0]
```

Comment:
`-fsycl-is-device` and `-fsycl-is-host` are `cc1` options and are not exposed to the user.  Maybe:  Set default value of `-sycl-std` to 2020 for SYCL enabled compilations.

Reply:
Thanks!